common-terms.md

```diff
@@ -0,0 +1,19 @@
+# Liste des termes communs
+
+De nombreux termes reviennent régulièrement au fil des pages et peuvent parfois poser un problème de traduction. En effet, il semble parfois plus judicieux de conserver une terminologie anglaise plutôt qu'une traduction absolue de chaque mot pouvant nuire à la compréhension.
+
+Cette liste est évolutive et constitue une simple proposition afin de conserver une cohérence d'ensemble, n'hésitez pas à la modifier pour rajouter ou corriger une traduction. 
+
+Vous pouvez également venir en discuter librement dans le salon dédié `localization-fr` sur le [Discord](https://gatsby.dev/discord)  
+
+| Terminologie originale      | Traduction proposée               |
+|---------------------|---------------------------|
+| Array               | Array                     |
+| Build               | Générer / Génération      |
+| Hot-reloading       | Rechargement instantané ? |
```

Comment:
If we don’t have a strong opinion about a term, maybe we should not putting it in this file. 

We could open an issue centralize the discussion or discuss it in discord. The advantage of the issue system is that it keeps things organized.

Reply:
Keeping track of this in an issue might be a good idea! 💪